projects/icons/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@gsa-sam/ngx-uswds-icons",
-  "version": "13.0.0-beta.2",
+  "version": "15.0.0",
   "types": "../../../dist/out-tsc/projects/icons/index.d.ts",
   "peerDependencies": {
-    "@angular/common": "~13.3.11",
-    "@angular/core": "~13.3.11"
+    "@angular/common": "^15.2.9",
+    "@angular/core": "^15.2.9"
   },
   "dependencies": {
-    "ngx-bootstrap-icons": "^1.7.2"
+    "ngx-bootstrap-icons": "^1.9.1"
```

Comment:
Do we know if the MFEs will be updating to node 16 with the ng15 update? ngx-bootstrap-icons>1.9 requires node 16 or greater, so this could cause an issue if the teams are not updating. I sent a message to Dave about this to see if he knows. I'll add another comment when I hear back from him.

Reply:
Teams already upgraded to node 18, but we will not merge this branch now as we need to support multiple versions till teams completely done angular 15 upgrade